src/node/connection_types.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
```

Comment:
I'm pretty sure Satoshi didn't write any of this code (same for other files)

Reply:
Done.